pkg/controllers/cloud/cloudprovideraccount_controller.go

```diff
@@ -83,7 +93,14 @@ func (r *CloudProviderAccountReconciler) Reconcile(ctx context.Context, req ctrl
 }
 
 func (r *CloudProviderAccountReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	var err error
 	r.accountProviderType = make(map[types.NamespacedName]common.ProviderType)
+	// Client in controller requires reconciler for each object that are under watch. So to avoid reconciler and use only
+	// watch, that can be implemented by clientset.
+	if r.clientset, err = kubernetes.NewForConfig(ctrl.GetConfigOrDie()); err != nil {
+		r.Log.Error(err, "error creating client config")
```

Comment:
creating config or just client?

Reply:
Bith right. Because ctr.GetConfigOrDie creates config and kubernetes creates clientset